### PR TITLE
bc: update Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
Hi, Thanks for your contribution. The page doesn't seem to be synced with https://github.com/tldr-pages/tldr/blob/main/pages/common/bc.md.

**Comment by kant** [comment]
you are right, tomorrow will be updated with the missing lines. Roger that!

File: pages.es/common/bc.md [hunk 1]
```diff
@@ -3,6 +3,14 @@
...
+
+`bc`
+
+- Inicia una sesión en modo interactivo with la biblioteca matemática estándar:
+
+`bc --mathlib`
+
 - Inicia `bc` en el modo interactivo utilizando la biblioteca matemática estándar:
 
 `bc -l`
```
Comment (EmilySeville7cfg) [inline pages.es/common/bc.md:16]:
```suggestion
- Inicia `bc` en el modo interactivo utilizando la biblioteca matemática estándar:

`bc --mathlib`
```

Mandatory fix: [long option must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! ❤️

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by kbdharun — COMMENTED** [review]
Still there seem to be some differences here with this page https://github.com/tldr-pages/tldr/blob/main/pages/common/bc.md. 

I have a question are you trying to translate the page in common or osx?

**Pull request closed without merging**